assets/style.css

```diff
@@ -37,10 +42,12 @@ iframe {
 }
 
 #site-footer {
+  width: 100%;
```

Comment:
for me width did not work to occupy the entire bottom which is why I had used max-width. Can you check this once, or may be include a screenshot? 

Otherwise rest looks good to me!

Reply:
Oh yeah my bad, I will take this out, the max width overrides it below so this is dead code